chrome/skin/default/zotero/report/detail_print.css

```diff
@@ -13,6 +13,9 @@ h1, h2, h3, h4, h5, h6 {
 
 ul, ol, dl {
 	page-break-inside: avoid;
+	-webkit-print-color-adjust: exact;
+	printer-colors: exact;
+	color-adjust: exact;
```

Comment:
We only need `color-adjust`, which is [supported in current browsers](https://developer.mozilla.org/en-US/docs/Web/CSS/color-adjust#Browser_compatibility). (The `-webkit` prefix is no longer required in Chrome, and as far as I know `printer-colors` was never in an actual spec.)

Reply:
Fixed.